functions/awsUploader/index.js

```diff
@@ -1,20 +1,35 @@
 const AWS = require("aws-sdk");
 const stream = require("stream");
+const fetch = require("node-fetch");
 
-const uploadS3Stream = ({ AWSConfig, Key, Metadata }) => {
-  const Bucket = AWSConfig.name;
-  const s3 = new AWS.S3({
-    region: AWSConfig.region,
-    accessKeyId: AWSConfig.key,
-    secretAccessKey: AWSConfig.secret,
+const getSignedUrl = async (params) => {
+  return new Promise((resolve, reject) => {
+    s3.getSignedUrl("putObject", params, (err, url) => {
+      if (err) {
+        console.log("err", err);
+        reject(err)
+      }
+
+      console.log("url", url);
+      resolve(url);
+    })
   });
+}
```

Comment:
this looks good - can it be moved to utils?

Reply:
done